ensembl/htdocs/info/genome/compara/species_trees.html

```diff
@@ -17,19 +17,72 @@ <h1 id="speciestrees">Species trees</h1>
 <ol>
     <li>The <a href="https://www.ncbi.nlm.nih.gov/taxonomy">NCBI taxonomy</a> (i.e. topology only) in:
     <ul>
-        <li>the <a href="/info/genome/compara/homology_method.html">Protein-trees</a> pipeline</li>
-        <li>the <a href="/info/genome/compara/ncRNA_methods.html">ncRNA-trees</a> pipeline, where the three sub-trees <em>Eutheria</em>, <em>Sauria</em>, and <em>Clupeocephala</em> are flattened</li>
-        <li>the CAFE pipelines (Gene Gain/Loss trees), with branch lengths coming from  <a href="https://www.timetree.org">the TimeTree database</a>
+        <li>the <a href="/info/genome/compara/homology_method.html">Protein-trees</a> pipeline for all divisions except Vertebrates</li>
+        <li>the CAFE (Gene Gain/Loss tree) pipelines in all divisions except Vertebrates</li>
```

Comment:
...and removed from here:
```suggestion
        <li>the <a href="/info/genome/compara/homology_method.html">Protein-trees</a> pipeline</li>
        <li>the CAFE (Gene Gain/Loss tree) pipelines</li>
```